guides/lambda/README.md

```diff
@@ -0,0 +1,57 @@
+# How to Set Up a Lambda
+
+Lambdas require some additional set up.
+
+## Creating the Lambda in AWS
+
+When you create a lambda that will be deployed with uDeploy you need to add versioning and create an alias for each environment.  The alias must point to a specific version.  If you point to $LATEST uDeploy will not work.  Additionally, you must use the lambda description field to hold your semantic version number.  
+
+### Terraform 
+
+To create an alias that points to a specifc version:
+```
+resource "aws_lambda_alias" "function_alias" {
+  name             = "${var.app}-alias-${var.environment}"
+  description      = "alias for udeploy"
+  function_name    = aws_lambda_function.function.arn
+  function_version = "1"
+}
+```
+
+### AWS Console 
+
+In this screenshot you can see that the lambda description is set to **1.0.0.20**.  This is the version that will be parsed by the regex in uDeploy.  You can put anything you want here as you are responsible for creating the regex.
+
+![Description](./console_description.png)
+
+In this screenshot you can see the alias **ams-aerial-lrc-lambda-dev-alias** is currently pointing to version 8.  Each time a new version of the lambda is published you must point your alias to the new version.  Pointing the alias to $LATEST will not work.
```

Comment:
"you must point your alias to the new version" Is this referring to the build process in dev? uDeploy will do this for you when deploying to each environment. Should that be specified?